daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -718,10 +718,16 @@ object SBuiltin {
   }
 
   /** $endExercise[T] :: Token -> () */
```

Comment:
Please update the comment for this exercise with the new parameter, similar to other `SBU*` case classes.

Reply:
Done